phoenix-core/src/it/java/org/apache/phoenix/end2end/MaxLookbackIT.java

```diff
@@ -168,36 +184,48 @@ public void testTTLAndMaxLookbackAge() throws Exception {
         conf.setLong(HRegion.MEMSTORE_PERIODIC_FLUSH_INTERVAL, 0L);
         try (Connection conn = DriverManager.getConnection(getUrl())) {
             String dataTableName = generateUniqueName();
-            String indexStem = generateUniqueName();
-            createTableAndIndexes(conn, dataTableName, indexStem);
-            long afterFirstInsertSCN = org.apache.phoenix.util.EnvironmentEdgeManager.currentTimeMillis();
+            createTable(dataTableName);
+            //increment by 10 min to make sure we don't "look back" past table creation
```

Comment:
Nit: consistent comment here

Reply:
WAIT_AFTER_TABLE_CREATION isn't meant to be a factor of MAX_LOOKBACK_AGE -- it's just an arbitrarily large number to make sure that all the metadata is older than the current timestamp. It could probably be lower, but since the size doesn't cost us anything I didn't spend too much time trying to optimize it.

Reply:
Turns out any value >= 1 works. I'll just switch it to 1

Reply:
The suggestion was not to have WAIT_AFTER_TABLE_CREATION be a multiple of MAX_LOOKBACK_AGE necessarily, but something larger than MAX_LOOKBACK_AGE  -- just something so that if you did (now - MAX_LOOKBACK_AGE) it didn't go before table creation time.

Reply:
That's not necessary because we always advance time by MAX_LOOKBACK_AGE before trying to look back that far.